.github/ISSUE_TEMPLATE/feedback_template.yml

```diff
@@ -0,0 +1,65 @@
+name: "Draft version 2 feedback"
+description: "Metadata model draft version 2 feedback form"
+title: "feedback: "
+labels: ["draft-v2-feedback"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Health-RI metadata model version 2 draft feedback
+
+        Please use this form to describe feedback related to the plateau 2 draft version of the core / generic health metadata schema (link).
```

Comment:
Link target needs to be decided, and properly encoded in markdown.

Reply:
Based on the discussion yesterday, link will be to develop branch.

https://github.com/Health-RI/health-ri-metadata/tree/develop